api/controllers/v1/slack.ts

```diff
@@ -142,7 +150,7 @@ export const getDeployableJobs = async (
       directory
     );
 
-    if (!aliases || aliases.length === 0) {
+    if (!aliases || aliases.length) {
```

Comment:
Do we want to keep the check for 0 aliases? Please correct me if I'm wrong, but I believe this change will prevent aliases from building

Reply:
yup this was an accident, thanks!